impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java

```diff
@@ -74,15 +74,7 @@ static Type getType(Measure measure, Aggregation aggregation) {
                 Functions.returnConstant(Type.GAUGE_DOUBLE), // LastValue Double
                 Functions.returnConstant(Type.GAUGE_INT64), // LastValue Long
                 Functions.<Type>throwAssertionError())),
-        new Function<Aggregation, Type>() {
-          @Override
-          public Type apply(Aggregation arg) {
-            if (arg instanceof Aggregation.Mean) {
-              return Type.CUMULATIVE_DOUBLE; // Mean
-            }
-            throw new AssertionError();
-          }
-        });
+        AGGREGATION_TYPE_DEFAULT_FUNCTION);
```

Comment:
You should make all the functions static if you want to have effect (e.g. Functions.returnConstant(Type.CUMULATIVE_DOUBLE) or Functions.returnConstant(Type.CUMULATIVE_INT64))

Reply:
I see, updated.